pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -1847,7 +1874,7 @@ func TestNodeShouldRunDaemonPod(t *testing.T) {
 				},
 			},
 			wantToRun:             true,
-			shouldSchedule:        false,
+			shouldCreate:          shouldCreate, //This is because we don't care about the resource constraints any more and let default
```

Comment:
I guess you have not finished the comment.

Reply:
Done. Thanks.